prowler/providers/aws/services/elasticache/elasticache_redis_replication_group_auth_enabled/elasticache_redis_replication_group_auth_enabled.py

```diff
@@ -0,0 +1,34 @@
+from packaging import version
+
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.elasticache.elasticache_client import (
+    elasticache_client,
+)
+
+
+class elasticache_redis_replication_group_auth_enabled(Check):
+    def execute(self):
+        findings = []
+        for repl_group in elasticache_client.replication_groups.values():
+            report = Check_Report_AWS(self.metadata())
+            report.region = repl_group.region
+            report.resource_id = repl_group.id
+            report.resource_arn = repl_group.arn
+            report.resource_tags = repl_group.tags
+
+            for cluster in repl_group.member_clusters:
```

Comment:
Please, can you show in the status extended the member clusters that are not compliant (if any).